maintainers/maintainer-list.nix

```diff
@@ -5224,6 +5224,12 @@
     githubId = 6074754;
     name = "Hlodver Sigurdsson";
   };
+  hufman = {
+    email = "hufman@gmail.com";
+    github = "hufman";
+    githubId = 1592375;
+    name = "Walter Huf";
+  };
```

Comment:
This needs to be sorted. Check comments on the top of the file.

Reply:
I understand this needs to be sorted. Is this the wrong place? I thought this order is correct: hlo .. huf .. hug

Reply:
Oh I see, hugoreeves is in the wrong place, I'll fix that.